src/OpenDB/test/gcd_abstract_lef.tcl

```diff
@@ -0,0 +1,17 @@
+source "helpers.tcl"
+
+# Open database, load lef and design
+
+set db [ord::get_db]
+read_lef "data/Nangate45/NangateOpenCellLibrary.mod.lef"
+read_def "data/gcd/floorplan.def"
+set chip [$db getChip]
+
+set lef_file [make_result_file gcd_abstract_lef.lef]
+
+set block [$chip getBlock]
+
+
+$block saveLef $lef_file
```

Comment:
You could just replace `$block` with `[ord::get_db_block]` and void having to get the `$db`, `$chip`, and `$block` separately.